src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStartedService.ts

```diff
@@ -190,7 +191,7 @@ export class WalkthroughsService extends Disposable implements IWalkthroughsServ
 							category: category.id,
 							order: index,
 							when: ContextKeyExpr.deserialize(step.when) ?? ContextKeyExpr.true(),
-							media: step.media.type === 'image'
+							media: step.media ? (step.media.type === 'image'
 								? {
 									type: 'image',
 									altText: step.media.altText,
```

Comment:
I think the `step.media` could throw on line 200?

```
step.media.type === 'svg'
```

Can we fix `step.media`'s type to prevent this?